gnosis/safe-react
# feat: Sidebar redesign

## What it solves
Resolves #3504 

## How this PR fixes it
- Adjusts the design according to [Figma](https://www.figma.com/file/Y9HUUDQzrbOtMpC3Nye3lt/Safe-Dashboard?node-id=997%3A3128).
- Adds Queue, History links to the navigation

## How to test it
1. Open the Safe app
3. Observe that there are links to the Transaction Queue and History
4. Observe that the Sidebar looks according to the design

## Screenshots
![Screenshot 2022-04-11 at 11 07 16](https://user-images.githubusercontent.com/5880855/162703523-24a6854b-17e1-4a68-b95b-eb4826e41088.png)
